.github/workflows/ci.yml

```diff
@@ -270,12 +331,13 @@ jobs:
       - name: Install wasm-pack
         run: curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh
 
-      - name: Run Wasm tests
-        run: wasm-pack test --chrome --headless
+      - name: Run Wasm tests (c2pa-rs)
```

Comment:
FWIW I removed the `(c2pa-rs)` label in my PR #1115 because there were no longer any other crates running Wasm or WASI.